Makefile

```diff
@@ -1,4 +1,7 @@
-check: fmt build test docs
+# Do test-loop last, since the tests for that target require sudo.
+# Using sudo changes permissions on various directories. It is less trouble
+# not to have to fix up permissions after every sudo'd test.
+check: fmt build test docs test-loop
```

Comment:
It strikes me as odd to have docs between test and test-loop. Can we move it before test, or is there a reason it's where it is?